README.md

```diff
@@ -66,34 +66,25 @@ A single build requires about 100 GiB of disk space. The cache directories in
 
 ## Building a BISDN Linux image
 
-### x86_64 platforms
+BISDN Linux supports two different architectures: ARM XGS iProc, and intel x86-64.
 
-The default image works on all x86_64 platforms supported by BISDN
-Linux. To build the image from the latest sources, clone this repo and
-run the following command in its root directory:
+To select the target machine for BISDN Linux pass the appropriate name via
+the`KAS_MACHINE` environment variable:
 
-```bash
-kas-container build bisdn-linux.yaml
-```
-
-After the build process has finished, your image will be in
-`build/tmp/deploy/images/generic-x86-64/`. A symlink named
-`onie-bisdn-full-generic-x86-64.bin` will point to the actual image
-file named `onie-bisdn-full-generic-x86-64-<timestamp>.bin`.
-
-### ARM platforms (Edgecore AS4610)
+* `generic-armel-iproc` for devices based on ARM XGS iProc (Accton AS4610 series)
+* `generic-x86-64` for devices with a x86 host CPU (all other supported devices)
 
-To build an image that works on all ARM platforms supported by BISDN Linux,
-run this command in the root of this repository:
+To build the image from the latest sources, clone this repo and run the
+following command in its root directory, e.g:
 
 ```bash
-KAS_MACHINE=generic-armel-iproc kas-container build bisdn-linux.yaml
+KAS_MACHINE=generic-x86-64 kas-container build bisdn-linux.yaml
 ```
 
-After the build process finishes, your image will be in
-`build/tmp/deploy/images/generic-armel-iproc/`. A symlink named
-`onie-bisdn-full-generic-armel-iproc.bin` will point to the actual image
-file named `onie-bisdn-full-generic-armel-iproc-<timestamp>.bin`.
+After the build process has finished, your image will be in
+`build/tmp/deploy/images/<KAS_MACHINE>/`. A symlink named
+`onie-bisdn-full-<KAS_MACHINE>.bin` will point to the actual image
+file named `onie-bisdn-full-generic-<KAS_MACHINE>-<timestamp>.bin`.
```

Comment:
```suggestion
file named `onie-bisdn-full-<KAS_MACHINE>-<timestamp>.bin`.
```

Reply:
If you want to drop the default machine, then there are more instances that need changing, e.g. in BUILDING_DETAILS.md.

Reply:
Did I miss any? I also updated BUILDING_DETAILS.md.

@jklare fixed.